src/GitVersionTask/GenerateGitVersionInformation.cs

```diff
@@ -1,42 +1,99 @@
 namespace GitVersionTask
 {
+    using System;
     using System.IO;
     using GitVersion;
     using GitVersion.Helpers;
-    using Microsoft.Build.Framework;
 
-    public class GenerateGitVersionInformation : GitVersionTaskBase
+    public static class GenerateGitVersionInformation
```

Comment:
@stazz, why the change to `static class` here?

Reply:
The reason is that the class has no state on its own anymore (instead, the `Input` and `Output` inner classes are used), so no instances of it will be generated either.

Reply:
I see. Thanks for the explanation.